Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "dbmeter"
+version = "0.1.0"
+authors = ["Anton Okolelov <okoanton@yandex.ru>", "Oleg Grizak"]
```

Comment:
Can you change last author name to `Oleg Gritsak` and add email `hack3p@gmail.com`?